packages/dids/.c8rc.json

```diff
@@ -14,6 +14,7 @@
   ],
   "reporter": [
     "cobertura",
+    "html",
```

Comment:
did you mean for this to happen?

Reply:
Yes: QoL - Added HTML code coverage output for `dids` repo for immediate coverage feedback.